josm.openstreetmap.de.yaml

```diff
@@ -0,0 +1,71 @@
+app-id: josm.openstreetmap.de
```

Comment:
this is not a valid app-id. for more on this take a look at the flatpak documentation. http://docs.flatpak.org/en/latest/conventions.html#application-ids

given that this is a java app it's also very likely to have it's own reverse dns app-id embedded throughout it already.

Reply:
what's wrong with the app id?  It's not obvious to me that the given id is not adhering to the reference you linked to.

Do you insist on reversing the elements because you assume a certain hierarchy in these bytes?

The upstream app id is "josm.desktop" which is hardly conforming to the `three-part form` mentioned in http://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
> Interface names should start with the reversed DNS domain name of the author of the interface (in lower-case), like interface names in Java

https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names

Reply:
upstream changed the appid :)